extension/intents/forms/formsContentScript.js

```diff
@@ -89,11 +89,7 @@ this.dictationContentScript = (function() {
     setTimeout(() => {
       if (el.hasAttribute("contenteditable")) {
         // eslint-disable-next-line no-unsanitized/property
-        el.innerHTML = insertString(
-          el.innerHTML,
-          quoteHtml(text),
-          el.selectionStart
-        );
+        insertStringContentEditable(quoteHtml(text), el);
```

Comment:
Generally quoting should happen right where the text is used, then someone can read insertion code and know that things are quoted properly. But in this case no quoting is necessary, since `createTextNode` will do the quoting. Probably if you try to insert `"` with this code you'll get `&quot;`

Reply:
I did not realize that. I have made the changes. And I had to remove the function because it was no longer being used